site/content/docs/cli/project-structure.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Project Structure
+weight: 3
+menu:
+  docs:
+    parent: "CLI"
+---
+
+This is how a typical project using the Checkly CLI is organized. Most files, directories and paths are conventions you
+can tweak to your liking using `import` / `require` and setting `glob` patterns.
+
+## Directories and files
+
+- `checkly.config.ts` - Global project and CLI configuration. This one is mandatory.
```

Comment:
Just seeing the `.ts` file, should we mention JS / TS in the opening and explain that we recommend TS?